dao/mongo_test.go

```diff
@@ -125,3 +125,15 @@ func TestUnitPatchPaymentsWithRefundPendingStatus(t *testing.T) {
 		So(err.Error(), ShouldEqual, "the FindAndModify operation must have a Deployment set before Execute can be called")
 	})
 }
+
+func TestUnitGetPaymentRefunds(t *testing.T) {
+	Convey("Get payment refunds by payment Id", t, func() {
+		cfg, _ := config.Get()
+		client = &mongo.Client{}
+		dao := NewDAO(cfg)
+
+		resource, err := dao.GetPaymentRefunds("id123")
+		So(resource, ShouldBeNil)
+		So(err.Error(), ShouldEqual, "the Find operation must have a Deployment set before Execute can be called")
```

Comment:
what does this mean?

Reply:
If i understand correct this is a check on error message that could possibly come from the mongo driver on executing this function.